GitCommands/Git/GitModule.cs

```diff
@@ -4058,6 +4066,23 @@ string GetDateParameter(string param, DateTime? date)
                     : "";
             }
         }
+
+        internal TestAccessor GetTestAccessor() => new TestAccessor(this);
+
+        internal readonly struct TestAccessor
+        {
+            private readonly GitModule _gitModule;
+
+            public TestAccessor(GitModule gitModule)
+            {
+                _gitModule = gitModule;
+            }
+
+            public GitArgumentBuilder UpdateIndexCmd(bool showErrorsWhenStagingFiles)
+            {
+                return _gitModule.UpdateIndexCmd(showErrorsWhenStagingFiles);
+            }
```

Comment:
[nit] this could be simplified to 
```
public GitArgumentBuilder UpdateIndexCmd(bool showErrorsWhenStagingFiles) => _gitModule.UpdateIndexCmd(showErrorsWhenStagingFiles);
```

Reply:
Than the same simplification could be used here as well, or am I wrong?
https://github.com/gitextensions/gitextensions/blob/e920a4916efaf0779491715fc07ee3825676e7ed/GitUI/Editor/FileViewer.cs#L1532-L1545
I recently added these in #6397, and re-used the same approach again.

To make things right I would send a dedicated PR to apply the same simplification to all 4 methods, since I don't want to mix changes relative to a different PR in here. Are you ok with that?

Reply:
Yes 👍

Reply:
I force pushed the change in `GitModule.cs` to this PR and opened #6433 for `FileViewer.cs`.